.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea
-composer.lock
```

Comment:
That is indeed true but consider this: when do tests run? 

When a developer of contributor makes a change. 

So if the build breaks and you test without a composer.lock file you don't know whether it is a dependency that causes the break (which one?!) or the change you just did. While with a fixed composer.lock it is unambiguous that it is your change that broke the build and saves time debugging.

In an ideal situation you should monitor when dependencies update within your given version range and if that happens run the tests after having ran `composer update`, or by using `composer require --no-update`